pkg/test/framework/logging.go

```diff
@@ -34,6 +34,7 @@ func init() {
 
 func configureLogging() error {
 	o := *logOptionsFromCommandline
-
+	// This is spammy, even for tests
```

Comment:
This comment refers to the current state, before this PR, right? It will make people confusing when reading it with a WarnLevel below?

Reply:
Its spammy at info level, which is why we make it warning